src/io.rs

```diff
@@ -6,13 +6,24 @@ use wasm_bindgen::prelude::*;
 #[cfg(not(target_arch = "wasm32"))]
 use std::io::{stdout, Write};
 
-use crate::SPEED_FACTOR;
+use crate::{SPEED_FACTOR, INSTANT_PRINT_LINES};
+
+use std::sync::atomic::{AtomicU32, Ordering};
+static COUNTER: AtomicU32 = AtomicU32::new(0);
 
 #[cfg(not(target_arch = "wasm32"))]
 pub async fn csleep(length: u64) {
     use std::time;
+
     let speed_factor = *SPEED_FACTOR.lock().await;
-    let sleep_length = time::Duration::from_millis((1.0 / speed_factor * length as f32) as u64);
+    let count = COUNTER.fetch_add(1, Ordering::Relaxed);
+    let sleep_length = if count < *INSTANT_PRINT_LINES.lock().await {
+        // If user passed `--instant-print-lines`, there should be
+        // no pauses in first `INSTANT_PRINT_LINES` number of lines
+        time::Duration::new(0, 0)
+    } else {
+        time::Duration::from_millis((1.0 / speed_factor * length as f32) as u64)
+    };
```

Comment:
Please also handle the `wasm32` case.

Reply:
Yes, `make web` and then launch a web server on `dist/`. You will have to install `trunk`: `cargo install trunk`. An easy web server you can also install with cargo: `cargo install miniserve`. Then do `miniserve dist/` and then just go there with your browser.

Reply:
I'm unable to get the web server running, either on my local macbook, or a remote RHL server. Both my fork, and a clone from your local version. It gives 404 for the .wasm and .css files.

Reply:
Try this: `miniserve dist/ --route-prefix genact --spa --index index.html`